docs/guides/advanced/authentication.md

```diff
@@ -305,7 +305,38 @@ However if the API provider deviates too far from the OAuth 2.0 specification it
     Coda doesn't currently support the older 1.0 or 1.0a versions of the OAuth specification. If you would like to connect to an API that only supports these versions of the standard please [contact support][support] so that we can continue to gauge interest.
 
 
-<!-- TODO: Progressive OAuth -->
+#### Incremental OAuth
+
+If your Pack includes a lot of features you may likewise need to request a lot of scopes during authentication, which can scare away new users. An alternative approach is to use incremental authorization, where you start out only requesting a small initial set of scopes and request more as the user starts using features that require them.
```

Comment:
Maybe also worth a section on the other core use case, which is that of a pack that's already live that needs more scopes in a new version. You can simply include the new scopes in the new version, and as described below, if the initial request fails, the user is prompted to reauth and try again.

Reply:
+1 - my bet is this will be more commonly relevant. How about having a paragraph above this like:

As your pack grows you may find that you need to request more OAuth scopes than you initially did when your existing users connected. Coda allows new scopes to be added to Pack OAuth settings in a non-breaking way: we don't prompt the user to re-authorize until they try to use a Pack feature that fails. Once that happens, we notice that the connection the user was using was created with a stale list of OAuth scopes and we prompt them to re-authenticate it to get your new scopes. 

This progressive expansion of OAuth scopes can also work on a per-formula basis... all of your stuff

Reply:
I love that language @patrick-codaio! Added almost verbatim.